lib/web_socket.rb

```diff
@@ -166,20 +166,19 @@ def send(data)
       end
       case @web_socket_version
         when "hixie-75", "hixie-76"
-          data = force_encoding(data.dup(), "ASCII-8BIT")
-          write("\x00#{data}\xff")
+          write("\x00#{data}\xff".force_encoding("ASCII-8BIT"))
```

Comment:
didn't really look into why this is ASCII (or read thru the posts you mentioned) - but should we not prefer UTF-8?